### PR TITLE
Translate feauture

### PR DESCRIPTION
**Pull request by Gorgias-JosephBongrand**
Added signup-2-fr.js for french handling of WF form on FR version. 
Updated logic in main.js to map files to right pages

### CODE REVIEW

File: src/js/signup-2-fr.js [hunk 1]
```diff
@@ -0,0 +1,1156 @@
...
+    event.preventDefault();
+    event.stopPropagation();
+    signupButton.next("div[id*='-message-container']").empty();
+
+    var form = $(this);
+    var fullnameStatus = getFieldStatusStorage(fullname_key);
+    var emailStatus = getFieldStatusStorage(email_key);
+    var passwordStatus = getFieldStatusStorage(password_key);
+    var status = "error";
+    var messaging = "Please check fields with error, then submit";
```
Comment (jumarcial) [inline src/js/signup-2-fr.js:1086]: Message still in english

File: src/js/signup-2-fr.js [hunk 2]
```diff
@@ -0,0 +1,1055 @@
...
+const company_domain_key = 'company_domain';
+const account_domain_key = 'account_domain';
+const TOKEN_KEY = 'x-account-manager-session-token';
+const classErrorField = 'field-error';
+const classWarningField = 'field-warning';
+const classValidField = 'field-valid';
+const classValidText = 'text-valid';
+const classErrorMessage = 'message-error';
+const classWarningMessage = 'message-warning';
+const btnWaitString = 'Please wait';
```
Comment (jumarcial) [inline src/js/signup-2-fr.js:15]: @Gorgias-JosephBongrand this need to be translated "Veuillez patienter"

File: src/js/signup-2-fr.js [hunk 3]
```diff
@@ -0,0 +1,1055 @@
...
+    if( prefilled == true) {
+      accountDomainPrefilled(response, verifyStatus);
+    }
+    else{
+
+      if(accountDomainField.val() === recommendedSubdomain){
+        verifyStatus = 'valid';
+      }
+      else{
+        verifyMessage = 'Domaine workspace non disponible. Recommandation : '+ recommendedSubdomain ;
```
Comment (jumarcial) [inline src/js/signup-2-fr.js:479]: "domaine workspace" is not clear. Let's replace by "URL indisponible. Essayez plutôt: "

File: src/js/signup-2-fr.js [hunk 4]
```diff
@@ -0,0 +1,1055 @@
...
+
+function accountDomainPrefilled(response, verifyStatus){
+
+  var domain_exist = response.domain_exist;
+  var account_domain = response.account_domain;
+  var verifyMessage = "";
+
+  // shouldn't happen since the cloud function is supposed to incremante until a available domain is found
+  if(domain_exist == true) {
+    verifyMessage = "Ce workspace est déjà pris. Veuillez en choisir un autre."
```
Comment (jumarcial) [inline src/js/signup-2-fr.js:497]: "Cet URL d'espace de travail est déjà utilisé."

File: src/js/signup-2-fr.js [hunk 5]
```diff
@@ -0,0 +1,1055 @@
...
+
+// used to verfiy the domain field is not empty and with the right format
+function companyDomainVerify(verifyStatus){
+  // resetFieldStatus(emailField);
+  var regexCompanyDomain = /^(?:(?:https?:\/\/)?(?:www\.)?)?[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}(?:\/[^\s]*)?$/;
+  var verifyMessage = "";
+  var verifyStatus = verifyStatus;
+  
+  if (!companyDomainField.val() || companyDomainField.val() === '') {
+    verifyMessage =  'Nous avons besoin de votre URL pour créer votre compte';
```
Comment (jumarcial) [inline src/js/signup-2-fr.js:528]: Nous avons besoin de l'URL de votre boutique en ligne pour créer votre compte

File: src/js/signup-2-fr.js [hunk 6]
```diff
@@ -0,0 +1,1055 @@
...
+      if (response.responseJSON) {
+        for (const field of Object.keys(response.responseJSON)) {
+          const error = response.responseJSON[field]
+          const inputField = $('input[name=' + field + ']')
+          var stringToCheck = "Le site de l'entreprise ou l'url du help desk est déjà lié à un espace de travail existant. Veuillez essayer une nouvelle URL ou vous connecter";
+          var errorCheck = JSON.stringify(error).includes(stringToCheck);
+
+          if(errorCheck == true){
+            handleFieldStatus(companyDomainField,verifyStatus,"Cet URL est déjà lié à un espace de travail existant. Veuillez essayer une nouvelle URL ou vous connecter");
+            handleFieldStatus(accountDomainField,verifyStatus,"Ce workspace est déjà pris. Veuillez en choisir un autre.");
```
Comment (jumarcial) [inline src/js/signup-2-fr.js:657]: Cet URL d'espace de travail est déjà utilisé. Veuillez en choisir un autre.

File: src/js/demo.js
Comment (jumarcial) [inline src/js/demo.js:1]: You've created the const "demoFrLeadFormID". Now you need to use it in the logic of the addEventListener "OnFormReady" and "onFormSubmitted in order to execute the same logic than with the const demoLeadFormId

File: src/js/demo.js [hunk 7]
```diff
@@ -1,4 +1,5 @@
 const demoLeadFormId = 'ef92ccce-92bd-4010-847a-793f56b6b353';
+const demoFrLeadFormID = "af1d8fe3-2a0d-4dc8-afb4-eb08b6741f79";
```
Comment (jumarcial) [inline src/js/demo.js:2]: demoFrLeadFormID -> demoFrLeadFormId 

In our "naming convention" of JS variables, we captitalize the firt letter of each word, except the first word

**Review by jumarcial — COMMENTED** [review]
I checked all the 3 files, @joseph please let me know once you've followed up with all my comments so I can do a last check before your merge the branch. thanks!

**Review by Gorgias-JosephBongrand — COMMENTED** [review]
Merging translation update

**Pull request merged**